ui/webui/webui-desktop

```diff
@@ -86,4 +86,9 @@ else
     sleep 3
 fi
 
-HOME="$BROWSER_HOME" $BROWSER http://127.0.0.90"$URL_PATH"
+trap 'kill $B_PID; exit 0' INT TERM
```

Comment:
The signal list is incomplete; you should at least also terminate on `EXIT`, but adding `QUIT` and `PIPE` can't hurt either.

Also, please drop the `exit 0` -- running into that trap *already* means that your script is exiting.